src/components/Header/Menu/Menu.test.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Menu from './Menu';
+
+const MenuData = [
+  {
+    label: 'Menu Item 1',
+    path: '/item1'
+  },
+  {
+    label: 'Menu Item 2',
+    path: '/item2'
+  }
+];
+
+/**
+ * @param {NodeList} nodeList
+ * @param {string} attributeName
+ * @param {string} key
+ * @description
+ */
+function validateElementAttributeValue(nodeList, attributeName, key) {
+  return Array.from(nodeList).some(
+    node => node.getAttribute(attributeName) === key
+  );
+}
+
+describe('Menu Component', () => {
+
+  test('Renders a Menu', () => {
+    const { container } = render(
+      <Menu menu={MenuData} />
+    );
+    expect(container.querySelector('.Menu').textContent).toBeTruthy();
```

Comment:
Can we use test IDs to determine if an element exists, instead of searching by its CSS class name?